objects/account_to_api_tokens_connection.go

```diff
@@ -1,23 +1,28 @@
+
 // Copyright ©, 2023-present, Lightspark Group, Inc. - All Rights Reserved
 package objects
 
+
+
+
 type AccountToApiTokensConnection struct {
 
-	// Count The total count of objects in this connection, using the current filters. It is different from the number of objects returned in the current page (in the `entities` field).
-	Count int64 `json:"account_to_api_tokens_connection_count"`
+    // Count The total count of objects in this connection, using the current filters. It is different from the number of objects returned in the current page (in the `entities` field).
```

Comment:
Ah formatter issues here too. You'll probably actually want this one. Good to install Go - https://go.dev/doc/install